### PR TITLE
LPS-155124 Filter System Fields in Form builder

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-155124

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
Just started reviewing :)

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ObjectField/ObjectField.js [hunk 1]
```diff
@@ -95,6 +95,9 @@ const ObjectField = ({
 				else if (relationshipType) {
 					return false;
 				}
+				else if (system) {
```
Comment (naraandrade) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ObjectField/ObjectField.js:98]: @danvb112 I think you could just add the check for the `system` value in the `else if` above :) 
```
else if(relationshipType || system) {
    return false;
}
```

**Comment by danvb112** [comment]
ci:test:sf

**Review by naraandrade — APPROVED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ObjectField/ObjectField.js (hunk 1 above)
Comment (danvb112) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ObjectField/ObjectField.js]: Thank you @naraandrade ! I hadn't noticed that! :)

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**